readme.md

```diff
@@ -2,29 +2,91 @@
 
 [![Build](https://github.com/Letterbook/Letterbook/actions/workflows/pull-request.yml/badge.svg?branch=main)](https://github.com/Letterbook/Letterbook/actions/workflows/pull-request.yml)
 
-Letterbook is intended to be a mastodon-compatible federated microblogging server. The project has three primary goals:
+Letterbook is a federated microblogging service, implementing ActivityPub. The project was started with the hope to make hosting a fediverse server a better and more sustainable experience. We also want to make it a safer and more inclusive space for the people who make the fediverse their social media home. Those are big aspirations, but we have some solid plans about how to start, and we would love more input about where to go from there.
 
-1. Functional compatibility with Mastodon, and Mastodon clients
-2. Reduced cost and complexity to deploy and operate
-3. More robust moderation tools
+## Features
```

Comment:
Might be useful for some of these sub-labels like "Moderation" or "Administration" or "Ops" to have Issue labels we link to from here.

Reply:
That's probably a good idea. We just need to actually create those issues first 😅